auth/auth.go

```diff
@@ -631,6 +631,9 @@ func (auth *Authenticator) UpdateUserEmail(u User, email string) error {
 		if err != nil {
 			return nil, err
 		}
+		currentUser.SetUpdatedAt()
+		currentUser.SetCreatedAt(currentUser.GetCreatedAt())
```

Comment:
I don't understand why the `Set(Get())` pattern is required. I think it's not nescessary to call `SetCreatedAt` in these places at all, since the time is already set.

Reply:
Good point, not sure why I thought this was needed. Will remove this now